### ISSUE TITLE
PoC: Dockerize wrapped model

### ISSUE THREAD

**Issue by bbroeksema** [nervosum/proof-of-concepts#2]
# Goal: 
Learn how to wrap model and wrapping code in distroless docker image

Starting point: model code of #1 :

```
/py-model
  /model     # model code
  /wrappper  # application code
  requirements.txt
  Dockerfile  # <- The PoC is about this file
```
Relevant links:
* https://docs.docker.com/develop/develop-images/multistage-build/
* https://github.com/GoogleContainerTools/distroless
* https://reproducible-builds.org/

# Definition of Done 
- [x] `docker build .` results in a minimal docker image with the model running. 
- [x] The build process (i.e. running pip install, copying files to the right location in the image) *also* runs in a docker image.

Dependencies: 
- #1

**Comment by GuidoTournois** [comment]
Do I understand correctly, that training the model is not in scope of this story? In other words, we assume the docker has access to a trained model (either by mounting a volume or retrieving it from a storage location) and that the model code merely acts as the interface with this model?

**Comment by bbroeksema** [comment]
> Do I understand correctly, that training the model is not in scope of this story? In other words, we assume the docker has access to a trained model (either by mounting a volume or retrieving it from a storage location) and that the model code merely acts as the interface with this model?

Yes, indeed! You can just reuse the directory from PoC 1 in this PoC to get started quickly -> literally copy/paste and add a Docker file to get started with the building process.
